README.md

```diff
@@ -6,6 +6,25 @@
 
 > This is a preview version of the updated React Native Tools extension. It's not recommended to use the preview version and the stable one at the same time.
 
+## React Native Tools Preview migration guide
+
+In the Preview version of the extension we changed debug types in debug configurations. The table below shows the difference between debug configuration of the stable and the preview version.
```

Comment:
```suggestion
In the Preview version of the extension we changed debug types in debug configurations. The table below shows the difference between the debug configurations of the stable and the preview version.
```

Reply:
I think we could just cut this top part all together. I think this line:
> In order to use the preview version of the extension it's required to add `-preview` suffix in `type` field of a debug configuration of the RNT stable extension.

Explains exactly the changes, and also what users need to do to switch their launch configs.